README.md

```diff
@@ -1077,6 +1081,16 @@ security:
 
 **NOTE:** The OIDC feature is currently in Alpha. Breaking changes may occur. Use this feature at your own risk.
 
+#### TLS Encryption
+Gatus supports basic encryption with TLS. To enable this, certificate files in PEM format have to be provided.
+The example below shows an example configuration which makes gatus respond on port 4443 to HTTPS requests.
+
+```yaml
+web:
+  port: 4443
+  cert_file: "server.crt"
+  key_file: "server.key"
```

Comment:
All configuration parameters with multiple words have said words split by `-` rather than `_`, could you update that here and on L190-L191?

I'd also prefer if it was called `certificate-file` instead of `cert-file`, and `key-file` should instead be `private-key-file`.

---

Finally, I wonder whether if it wouldn't be a good idea to just directly specify the certificate and private key directly in the configuration... Yes, I know it's a little chaotic, but that'd make things a lot easier for people.

Something like:
```yaml
web:
    private-key: |
      -----BEGIN RSA PRIVATE KEY-----
      ...
      -----END RSA PRIVATE KEY-----
    certificate: |
      -----BEGIN CERTIFICATE-----
      ...
      -----END CERTIFICATE-----

```

Reply:
I really would not mix credentials and configuration

Reply:
The naming is adjusted as suggested. It really looks clearer now.